src/StoreApi/Utilities/OrderController.php

```diff
@@ -132,7 +149,12 @@ public function sync_customer_data_with_order( \WC_Order $order ) {
 					'shipping_phone'      => $order->get_shipping_phone(),
 				]
 			);
+			$order_fields = $this->additional_fields_controller->get_all_fields_from_order( $order );
 
+			$customer_fields = $this->additional_fields_controller->filter_fields_for_customer( $order_fields );
+			foreach ( $customer_fields as $key => $value ) {
+				$this->additional_fields_controller->persist_field_for_customer( $key, $value, $customer );
```

Comment:
Does this persist or just set? Because there is a `$customer->save();` below.

Reply:
Good point, I only set, we don't save.

Well, currently we also persist to customer due to a bug, but it should be set.

Reply:
Perhaps change method name to reflect this then.